grails-app/domain/org/pih/warehouse/inventory/OutboundStockMovementListItem.groovy

```diff
@@ -124,7 +125,16 @@ class OutboundStockMovementListItem implements Serializable {
     }
 
     Integer getLineItemCount() {
-        return requisition ? RequisitionItem.countByRequisition(requisition) : order ? OrderItem.countByOrder(order) : 0
+        if (requisition) {
+            RequisitionItem[] approvedItems = requisition.requisitionItems.findAll{ it ->
```

Comment:
I'm pretty sure findAll would return a List if you invoked it on a List (I'm almost positive that requisitionItems is a List) Does this work actually work? Does it work because of type coercion?

Reply:
yeah, it works, because `requsitionItems` is a list of `RequsitionItem`'s elements. I attach a screen below:
[screen](https://i.imgur.com/6LKeqE5.png)